### ISSUE TITLE
can't install ??

### ISSUE THREAD

**Issue by geomagilles** [BlueHotDog/sails-migrations#45]
When running 'sudo npm install sails-migrations -g': 

`````
npm ERR! Error: ENOENT, chmod '/usr/lib/node_modules/sails-migrations/bin/cli.js'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.13.0-30-generic
npm ERR! command "/usr/bin/node" "/usr/bin/npm" "install" "sails-migrations" "-g"
npm ERR! cwd /home/vagrant/sails
npm ERR! node -v v0.10.32
npm ERR! npm -v 1.4.28
npm ERR! path /usr/lib/node_modules/sails-migrations/bin/cli.js
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! not ok code 0
`````
Note: I run this within a vagrant box where sails is up and running

**Comment by BlueHotDog** [comment]
Fixed, try now.

**Comment by geomagilles** [comment]
it works, thanks

**Comment by geomagilles** [comment]
Installation works, but after global *and* local installation, when running 'sails-migrations generate add_user_table', I obtain:

`````
No local sails_migrations install found in: ~/sails/server

/usr/lib/node_modules/sails-migrations/bin/cli.js:40
    exit('Try running: `npm install sails-migrations`');
    ^
ReferenceError: exit is not defined
    at initSailsMigrations (/usr/lib/node_modules/sails-migrations/bin/cli.js:40:5)
    at Command.<anonymous> (/usr/lib/node_modules/sails-migrations/bin/cli.js:69:17)
    at Command.listener (/usr/lib/node_modules/sails-migrations/node_modules/commander/index.js:282:8)
    at Command.emit (events.js:98:17)
    at Command.parseArgs (/usr/lib/node_modules/sails-migrations/node_modules/commander/index.js:524:12)
    at Command.parse (/usr/lib/node_modules/sails-migrations/node_modules/commander/index.js:410:21)
    at Liftoff.invoke (/usr/lib/node_modules/sails-migrations/bin/cli.js:163:13)
    at Liftoff.launch (/usr/lib/node_modules/sails-migrations/node_modules/liftoff/index.js:163:8)
    at Object.<anonymous> (/usr/lib/node_modules/sails-migrations/bin/cli.js:178:5)
    at Module._compile (module.js:456:26)
````

**Comment by joaosa** [comment]
The new release appears to be working fine for me

**Comment by joaosa** [comment]
I did some digging and it appears the sails-migrations' configPath is not being passed correctly to node-liftoff.

If I explicitly set the 'sailsMigrationFile' arg, I get a different error (error: unknown option `--sailsMigrationFile'), but my sails configuration is detected correctly.

I can't seem to find the place where the said arg should be set. Could this be related to the issue?

**Comment by BlueHotDog** [comment]
ya, seems like a bug in liftoff, looking into it.

**Comment by BlueHotDog** [comment]
fixed all issues in 2.0.5 works for me locally, give it a go please.

**Comment by geomagilles** [comment]
seems to work - thanks
